gettingstarted/InstallingEquella.md

```diff
@@ -3,39 +3,47 @@
 # Installing openEQUELLA
 
 ## Environment setup
-Assuming you are building and installing openEQUELLA in the same environment, the following should be setup.  Details can be found in the [openEQUELLA Dev Setup](https://github.com/equella/Equella#download-required-software) guide.
+The following dependencies are required to successfully install openEQUELLA.
 
-- Java 8 (Oracle)
-- ImageMagick
-- Libav
-- Database
+- [Java 8 JDK](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) (Latest version)
 
-Needed if you are building your own install binary:
-- GIT
-- SBT
+- [ImageMagick](https://www.imagemagick.org/script/download.php) (Tested with ImageMagick 6.9.8 Q8)
 
-## Build the Installer
+- [LibAV](https://libav.org/download/) (tested with 11.7)
 
-If you don't have / don't want to use a pre-built binary, you can build openEQUELLA from source yourself by following the instructions on the main [openEQUELLA repo](https://github.com/equella/Equella).
+  (**Note:** This is not a compulsory step if video file attachments will not be used in your institution or you don’t require them to display in the Videos gallery view.)
```

Comment:
FWIW, and per #40 and #20 ; you should probably update this to say ffmpeg 3.2.

Reply:
Noted in #66.